python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/pyproject.toml.tmpl

```diff
@@ -1,7 +1,25 @@
+[project]
+name = "dagster_quickstart"
```

Comment:
```suggestion
name = "{{ code_location_name }}"
```
Would this enable dynamic substitution of the dagster project name?

So that `dagster_university_start` for example would be sustituted when makeing a project from the template.

Reply:
ah -- good catch